src/kedro_templar/process_templates.py

```diff
@@ -99,30 +40,24 @@ def download_file_from_s3(file_name, bucket):
     default=False,
     type=bool
 )
-@click.option(
-    "-u",
-    "--upload",
-    "upload_source",
-    required=False,
-    default=False,
-    type=bool
-)
 def process_templates(
     config_path: Text,
     templates_dir: Text,
     output_dir: Text,
     replace_existing: bool,
     upload_source: Text
 ):
-
-    config = load_config( s3_config if s3_config else config_path)
-    environment = create_environment(templates_dir)
+    # extend -c to accept also s3 path, @WOJTEK what do you think?
+    if str(config_path).startswith('s3://'):
```

Comment:
This kind of logic is acceptable, but only if hidden/wrapped into an utils functions.
Either way, we want to have a separate column for uploading/downloading files to/from S3

Reply:
changed implementation